daily-check.md

```diff
@@ -2,6 +2,7 @@
 title: 日常巡检
 summary: 介绍 TiDB 集群需要常关注的性能指标。
 category: reference
+aliases: ['/docs-cn/dev/daily-inspection/']
```

Comment:
Please s/dev/stable in 4.0 cp PR @toutdesuite

Reply:
updated in https://github.com/pingcap/docs-cn/pull/3613